### ISSUE TITLE
Yoko Oh Noes has a beef with literal arrays as arguments to println

### ISSUE THREAD

**Issue by lgto4** [Khan/live-editor#542]
`println([3, 2, 1]);`
produces the complaint "t looks like you're trying to use println(). In case you forgot, you can use it like: println(data) | ProcessingJS".

**Comment by DozenalFTW** [comment]
It sees a `,` and thinks you're passing a second argument to `println`, like `println(42, 21)`. I notice that using a comma in a string doesn't do _this_ bug, but I think the hint that appears guiding you through the text function bugs out.

**Comment by kevinbarabash** [comment]
That makes sense.  We do a lot of things with regexes that we should probably be doing with an AST.

**Comment by codeHusky** [comment]
Is this still an issue?

**Comment by lgto4** [comment]
Yes, the one line program still fails in the manner originally described.
